docs/src/tut/nft/index.md

```diff
@@ -0,0 +1,739 @@
+Don’t worry if you don’t understand what everything does in the solution. 
+This is expected! 
+
+After you feel comfortable with the basics, 
+you can head over to the detailed Reach [tutorials](https://docs.reach.sh/tut/#tuts) and become an expert in the Reach language.
+
+All of the code for this guide is [open source](https://github.com/reach-sh/reach-lang/tree/master/examples/nft-auction-api). 
+To get the most out of this guide, 
+we recommend reproducing the code in a [text editor](https://code.visualstudio.com/Download). 
+
+When learning a new language, 
+it is a best practice to type the code yourself and flip between windows, 
+rather than splitting the screen. 
+While this process is not as efficient, 
+it is much more effective in forcing you to think about the code as you recreate it.
+
+# Organization
+
+This guide is organized into four sections: 
+1. A summary of [Reach Architecture](#reach-architecture)
+1. An [install guide](#install-reach) to get up and running with Reach. 
+1. An [overview](#application-overview) of the Reach DApp.
+1. A step by step guide and explanation of components used while [building](#building-a-dapp) a Reach application.
+
+# Reach Architecture
+
+Using a high-level language such as Reach to build DApps instead of low-level assembly language is attractive for a number of reasons. 
+First, abstraction is a developer's friend. 
+Abstraction allows developers to focus on what really matters in their application, 
+such as, codifying the business logic of their program, 
+rather than determining where the application should access memory.
+
+Another reason is that most humans, even most engineers, 
+are not good at writing low-level code that is efficient, secure, and free of flaws. 
+Reach abstracts contract storage, protocol diagrams, state validation and network details so you can focus on how participants, users of the DApp, interact with one another and the contract.
+
+Reach's backend is very often one single Reach `rsh` file, which is syntactically the same as a JavaScript file, 
+but contains web3 functionality.
+The frontend in a Reach DApp can be created with JavaScript using Reach's [Standard JavaScript Library](https://docs.reach.sh/frontend/#ref-frontends).
+Alternatively, developers can use Python, Go, or C# in their frontend by utilizing Reach's [RPC Server](https://docs.reach.sh/rpc/).
+
+Reach's backend, often named `index.rsh`, 
+instructs data to be published to the consensus network (a blockchain) and defines interfaces that can interact with the frontend. 
+This interface is commonly referred to as a Participant Interact Interface. 
+The frontend defines the interact objects and is often contained within a single file called `index.mjs`. 
+The frontend provides a graphical representation of the participants, e.g. Alice and Bob. 
+The user interface may be in the form of a command line, 
+or more preferably, a web or mobile app.
```

Comment:
It doesn't always provide a graphical representation. I'd say it provides an interface to the participants